packages/web3/src/wagmi.ts

```diff
@@ -0,0 +1,28 @@
+// Centralized re-exports for the subset of Wagmi helpers we use across the monorepo.
+// ‑ We explicitly list each symbol to avoid duplicate exports between
+//   "wagmi" and "wagmi/actions", which was triggering ambiguous namespace
+//   resolution warnings in the build.
+// ‑ Feel free to extend this list as new helpers are required.
+
+// Core Wagmi hooks & utilities
+export {
```

Comment:
i don't understand why that's necessary, can't we export everything from wagmi instead of having to manage this whitelist manually?

Reply:
gotcha, thx for explaining. and why do we need `wagmi` as dependencies in the apps at all then if we pull its exports from `@web3`?